app/manifest.webapp

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "0.1.0",
+  "name": "openmrs-owa-platformui",
+  "description": "A very simple UI for the OpenMRS Platform to give access to the Addon Manager, User manager, and other tools.",
+  "launch_path": "index.html",
+  "icons": {
+    "48": "/img/omrs-button.png"
+  },
+  "developer": {
+    "name": "charpell",
+    "url": "https://github.com/charpell/openmrs-owa-openmrs-owa-platformui"
```

Comment:
Can we change this url to that of openmrs?

Reply:
I have updated the PR